src/sentry/testutils/helpers/redis.py

```diff
@@ -1,11 +1,20 @@
+from contextlib import contextmanager
 from typing import Any
 
 
-def get_redis_cluster_default_options(id: str, high_watermark: int = 100) -> dict[str, Any]:
-    return {
+@contextmanager
+def use_redis_cluster(
+    cluster_id: str = "cluster", high_watermark: int = 100, with_settings: dict[str, Any] = None
```

Comment:
this should be `with_settings: dict[str, Any] | None = None` -- I think this file might be excluded from type checking?

Reply:
Yeah you're right.